src/Http/Controllers/StaticPagesUserController.php

```diff
@@ -8,21 +8,21 @@
 class StaticPagesUserController extends Controller
 {
 
-    public function pageList()
+    public function pageList(StaticPage $staticPages)
     {
-        if (view()->exists('packages.static-pages.pages-list')) {
-            return view('packages.static-pages.pages-list');
-        } else {
-            return view('staticPages::site.pages-list');
-        }
+        $pages = $staticPages->where('status', $staticPages::STATUS_ACTIVE)->orderBy('updated_at', 'desc')->get();
+
+        return view('static-pages::site.pages-list', compact('pages'));
     }
 
-    public function pageView()
+    public function pageView(StaticPage $staticPages, $slug)
     {
-        if (view()->exists('packages.static-pages.pages-view')) {
-            return view('packages.static-pages.pages-view');
-        } else {
-            return view('staticPages::site.pages-view');
+        $page = $staticPages->where('status', $staticPages::STATUS_ACTIVE)->where('slug', $slug)->get()->first();
```

Comment:
Why do you use get() and first() together?
You have to use only one of them.
In this case it's first()